SQL-MYSQL/gestao_produtos/atualizar_registros_tabela_produtos.sql

```diff
@@ -0,0 +1,21 @@
+use gestao_produtos;
+
+update produtos
+set preco = 15.00, quantidade_estoque = 150
+where id = 1;
+
+update produtos
+SET preco = 25.00, quantidade_estoque = 250
+where id = 2;
+
+update produtos
+set preco = 45.00, quantidade_estoque = 60
+where id = 8;
+
+update produtos
+set id = 15
+where id = 21;
+
+update produtos
+set id = 10
+where id = 15;
```

Comment:
Esse tipo de manipulação não pode ser feita. Mantenha o ID sempre permanente e manipule apenas os dados. Quando for trabalhar com relacionamentos, onde as tabelas se comunicam entre si, esse ID será a ponte entre essas tabelas.

Exemplo. Vamos supor que você tem uma tabela chamada categoria, onde cada ID representa uma categoria específica.
```code
tabela_categoria
- id
- nome_categoria
```
Supondo que a categoria "Organicos" o ID dele é 1.

Na tabela de produtos vc relaciona seu produto a uma categoria qualquer
```code
tabela_produto
- id
- nome_produto
- id_categoria
```
Daí vc salva um produto organico dizendo que a categoria_id é 1

Se vc altera o ID da categoria na tabela_categoria, vc vai ter problemas pq o produto nao vai achar a categoria.